website/docs/r/tm_content_library.html.markdown

```diff
@@ -37,7 +37,7 @@ resource "vcd_tm_content_library" "cl" {
 The following arguments are supported:
 
 * `name` - (Required) The name of the Content Library
-* `storage_policy_ids` - (Required) A set of [Region Storage Policy](/providers/vmware/vcd/latest/docs/resources/tm_region_storage_policy) IDs or VDC Storage Policy IDs used by this Content Library
+* `storage_class_ids` - (Required) A set of [Region Storage Policy](/providers/vmware/vcd/latest/docs/resources/tm_region_storage_policy) IDs or VDC Storage Policy IDs used by this Content Library
```

Comment:
Double checking. Is this really still true. Is it not only region storage policy IDs?

Reply:
Yep, forgot to change. Put here the same description as API docs, with a reference to the data source